src/core/utils.js

```diff
@@ -354,9 +355,14 @@ export function isCompatibleWithUserAgent({
     // confused by this as tofumatt was.
     // See: https://github.com/mozilla/addons-frontend/issues/2074#issuecomment-286983423
     if (maxVersion && mozCompare(browser.version, maxVersion) === 1) {
-      _log.info(oneLine`maxVersion ${maxVersion} for add-on lower than browser
-        version ${browser.version}, but add-on still marked as compatible
-        because we largely ignore maxVersion. See:
+      // See: https://github.com/mozilla/addons-server/issues/5277
```

Comment:
I don't think this bug is relevant because `current_version.is_strict_compatibility_enabled` is not a new concept (it's just newly exposed in the API). The bug you linked to was just about automatically setting a bunch of add-ons to use strict compatibility. Developers can still manually turn on strict compatibility.